msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -137,6 +137,38 @@ The query parameters are:
 
 To use this deep link with your bot, you can specify this as the URL target in your card's button or tap action through the `openUrl` action type.
 
+## Generate deep links to file in channel
+
+The following deep link format can be used in a bot, connector, or messaging extension card:
+
+`https://teams.microsoft.com/I/file/5E0154FC-F2B4-4DA5-8CDA-F096E72C0A80?tenantId=<tenantid>&fileType=<filetype>&objectURL=<objectURL>&baseUrl=<baseURL>&serviceName=<Name>&threadId=<threadid>&groupID=<GroupId>`
+
+The query parameters are:
+
+* `tenantId`: Tenant ID example, 0d9b645f-597b-41f0-a2a3-ef103fbd91bb
+* `filetype`: Supported file type, such as Word, PPT, PDF, and Excel
+* `objectUrl`: Object URL of the file, https://microsoft.sharepoint.com/teams/(filepath)
+* `baseUrl`: Base URL of the file, https://microsoft.sharepoint.com/teams
+* `serviceName`: Name of the service, Teams
```

Comment:
We should probably just ask app developers to put their app ID as the `serviceName` so that we have a consistent way to know where these links are coming from